src/client.js

```diff
@@ -98,6 +98,14 @@ export function delContainer(system, id, force) {
     });
 }
 
+export function renameContainer(system, id, config) {
+    return new Promise((resolve, reject) => {
+        podmanCall("libpod/containers/" + id + "/rename", "POST", config, system)
+                .then(resolve)
+                .catch(reject);
+    });
+}
```

Comment:
I know this is just copying the previous pattern, but that Promise() wrapper is completely useless. Please use something like this:

```js
export const renameContainer = (system, id, config) => podmanCall("libpod/containers/" + id + "/rename", "POST", config, system);
```

or if you prefer, just change the function body to
```js
return podmanCall("libpod/containers/" + id + "/rename", "POST", config, system);
```

Reply:
Thanks @martinpitt for the review, I will fix the code and prepare the tests when I have some free time to code. In my case it's usually during the holidays. ;-)